src/types/onyx/Report.ts

```diff
@@ -37,11 +37,14 @@ type PendingChatMember = {
 
 /** Report participant properties */
 type Participant = OnyxCommon.OnyxValueWithOfflineFeedback<{
+    /** Whether the participant is hidden */
+    hidden?: boolean;
```

Comment:
Why are we adding all this to the report type? We recently worked in this issue to clean up the Report type to only store data that is sent by the server. https://github.com/Expensify/App/issues/51867

Reply:
I added this because it's being returned from the server 😅 

<img width="328" alt="Screenshot 2024-11-14 at 18 16 19" src="https://github.com/user-attachments/assets/518288ef-8ff6-4900-81ff-3690c2fab3a0">

Reply:
Ah gotcha. Yeah "hidden" is being removed from the server as we speak. What about the other ones. Do we need those?

Reply:
I just removed all the unused properties like agreed in https://github.com/Expensify/App/issues/51867 😄 

Thanks for the feedback @puneetlath 😄